test/functional/Java8andUp/testng.xml

```diff
@@ -30,6 +30,7 @@
 	<test name="generalTest">
 		<classes>
 			<class name="org.openj9.test.gpu.SortTest" />
+			<class name="org.openj9.test.truncatedReturn.TestTruncatedReturn" />
```

Comment:
We are planning to rename the generalTest to GPUSortTest to be more straightforward. Each testname could indicate kind of its feature just like all other tests. So probably adding a new test naming `truncateReturn`  would be better.